### PR TITLE
Support disabling SSL verification

### PR DESCRIPTION
**Pull request by GuyKh**

### CODE REVIEW

File: glances_api/__init__.py [hunk 1]
```diff
@@ -31,6 +32,7 @@ def __init__(
         self.values = None
         self.username = username
         self.password = password
+        self.verify_ssl = verify_ssl if verify_ssl else True
```
Comment (fabaff) [inline glances_api/__init__.py:35]: `glances` has no support for HTTPS out-of-the-box. This is only relevant if one is running `glances` behind a reverse proxy with a self-signed certificate. I guess that not many installation have used this kind of setup over the last six years while the integration was available.

Has `glances` support for HTTPS now?

Comment (GuyKh) [reply glances_api/__init__.py:35]: I'm not sure... This was originally made for supporting the current 'verify_ssl' boolean setting as well

Comment (fabaff) [reply glances_api/__init__.py:35]: The conditional seems superfluous as there is a default.

```suggestion
        self.verify_ssl = verify_ssl
```

**Review by fabaff — COMMENTED** [review]
The test needs an update.

**Comment by GuyKh** [comment]
> The test needs an update.

Not sure why - since it has a default value - why would that make a difference?

**Comment by fabaff** [comment]
> Not sure why - since it has a default value - why would that make a difference?

The test and the example are failing as `verifiy` is implemented in the wrong way.
 
```bash
[...]
  File "/home/fab/Documents/repos/python-glances-api/example.py", line 15, in main
    await data.get_metrics("mem")
  File "/home/fab/Documents/repos/python-glances-api/glances_api/__init__.py", line 71, in get_metrics
    await self.get_data("all")
  File "/home/fab/Documents/repos/python-glances-api/glances_api/__init__.py", line 43, in get_data
    response = await client.get(str(url), verify=self.verify_ssl)
TypeError: AsyncClient.get() got an unexpected keyword argument 'verify'
````

See https://www.python-httpx.org/advanced/#ssl-configuration-on-client-instances

**Pull request merged**